website/versioned_docs/version-7.0.0/traverse.md

```diff
@@ -33,5 +33,17 @@ traverse(ast, {
   }
 });
 ```
+
+Also, we can target particular [**node types**](https://babeljs.io/docs/en/babel-types#api) in the Syntax Tree
+
+```js
+traverse(ast, {
+    FunctionDeclaration: function(path) {
+             path.node.name = "x";
```

Comment:
FunctionDeclaration nodes don't have a `name` property; could you change this to Identifier or change the code?

Reply:
Sorry. Id property of the FunctionDecleration node has a name.  I missed. I'm on it.

Reply:
I fixed it. Could you please check...